test/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE HTML>
 <html>
   <head>
-    <title>Vega Theme Test</title>
-    <script src="../node_modules/vega-lib/build/vega.js"></script>
-    <script src="../node_modules/vega-lite/build/vega-lite.js"></script>
-    <script src="../node_modules/vega-embed/build/vega-embed.js"></script>
-    <script src="../build/vega-themes.js"></script>
+    <title>Vega Theme Collections</title>
+    <script src="https://cdn.jsdelivr.net/npm/vega@4"></script>
+    <script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
```

Comment:
Unfortunately, this is not an option for us as we often want to use a local build of these libraries. By using jsdelivr, we won't be able to that.

Reply:
then we have to push the node_modules in the reop?

Reply:
No, we do not. We could have a separate index.html for the website for example. Or we could build a bundle (that's a lot of overhead, though).

Reply:
what should I do now? And how I can push seperate index.html to gh-page branch.

Reply:
Oh, I was imagining a separate folder `dist` where you have an `index.html` file that uses jsdelivr instead of local files.